packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -639,6 +645,25 @@ export default {
             });
             return;
           }
+          if (useEventVariables.has(declaredDependencyNode)) {
+            reportProblem({
+              node: declaredDependencyNode,
+              message:
+                '`useEvent` functions always return a new identity for every render. This means ' +
```

Comment:
I feel like this message implies that anything with always-new identity should be excluded. But this is only true for useEvent specifically. I'd maybe just focus on the recommendation: `Functions wrapped in useEvent must not be included in the dependency array. Remove 'onFoo' from the list.`

Reply:
Maybe `Functions returned from`? It's not the inner function that can't be depended on.

Reply:
Done, thanks both!